my-app/src/MapView.jsx

```diff
@@ -51,13 +52,18 @@ const MapView = () => {
     }
   };
 
-  const markersOverLap = (marker1, marker2) => {
-    const latDiff = marker1.position.lat - marker2.position.lat;
-    const lngDiff = marker1.position.lng - marker2.position.lng;
-    return (
-      latDiff * latDiff + lngDiff * lngDiff < overlapRadius * overlapRadius
-    );
-  };
+  function markersOverLap(marker1, marker2) {
+    console.log(marker1);
+    console.log(marker2);
+    var p1 = mapRef.current.getProjection().fromLatLngToPoint(marker1.position);
+    var p2 = mapRef.current.getProjection().fromLatLngToPoint(marker2.position);
+    var pixelSize = Math.pow(2, -mapRef.current.getZoom());
+
+    var d =
```

Comment:
What does d represent? What would have been a better variable name for this?

Reply:
D represents distance ! I would change that variable name to make it clear